gsoc/2024/mentors.md

```diff
@@ -17,6 +18,7 @@ layout: plain
 * David Lange [david.lange@cern.ch](mailto:david.lange@cern.ch) CompRes
 * Wim Lavrijsen [wlavrijsen@lbl.gov](mailto:wlavrijsen@lbl.gov) CompRes
 * Thomas Madlener [thomas.madlener@cern.ch](mailto:thomas.madlener@desy.de) DESY
+* Johan Mabille [johan.mabille@quantstack.net](mailto:johan.mabille@quantstack.net) DESY
```

Comment:
QuantStack instead of DESY
Otherwise looks good

Reply:
Good catch!